### ISSUE TITLE
Forge Hammer continues to be problematic

### ISSUE THREAD

**Issue by Parker8283** [WanionCane/UniDict#90]
Seems like a never-ending cycle with this thing :P

Using a slightly modified version of the Direwolf20 pack, the following plates are uncraftable with the Forge Hammer:
* Gold
* Copper
* Steel

So I'm not 100% sure if all the other metal plates from Thermal Foundation should be craftable with the Forge Hammer naturally or not, so if so, a lot more exist too. Something else that I have noticed is that plates that I *can* craft with the Forge Hammer, I can't craft them with Immersive Engineering's Engineer's Hammer. The other way around is true as well.

* Forge 14.23.2.2627
* UniDict 2.5

**Comment by WanionCane** [comment]
oh no.. not again XD
so... I will take a look... again...

**Comment by FUZZYMANBOOB98** [comment]
Still a problem in latest version. Is there something in the config I can mess with to get this working myself rather than hoping something doesn't break it every other update?

I know there's an ignore recipe option in the config but it's not clear how to use it and the format. I tried adding IC2:iron_plate for instance in ignored recipes but it didn't seem to make any difference.

I should mention the version was 2.5f. I rolled back each version to 2.3 and none of them worked. Uninstalling UniDict, WanionLib and keeping only InstantUnify + CraftTweaker made the problem go away. I'd like to be able to use UniDict though because it's ease is just stellar.
